### DISCUSSION TITLE
xrdp_wm_log_msg: Can't create session for user

### DISCUSSION THREAD

**Discussion by 720lhy** [neutrinolabs/xrdp#3331] [Q&A]
ubuntu22.04.1  xrdp0.10.1  Help! Has anyone experienced this problem? The xsession cannot be created

12月 05 10:57:35 xrdp[88005]: [INFO ] xrdp_wm_log_msg: sesman connect ok
12月 05 10:57:35 xrdp[88005]: [INFO ] xrdp_wm_log_msg: Performing login request for liaohaoyu
12月 05 10:57:36  xrdp[88005]: [INFO ] xrdp_wm_log_msg: login was successful - creating session
12月 05 10:57:46  xrdp[88005]: [INFO ] xrdp_wm_log_msg: Can't create session for user lxxx - X server could not be started
12月 05 10:59:18 xrdp[88005]: [INFO ] Login screen monitor height is 2160 pixels over 335 mm (163 DPI)
12月 05 10:59:27 xrdp[88005]: [ERROR] xrdp_sec_recv: xrdp_mcs_recv failed
12月 05 10:59:27  xrdp[88005]: [ERROR] xrdp_rdp_recv: xrdp_sec_recv failed
12月 05 10:59:27  xrdp[88005]: [ERROR] libxrdp_process_data: xrdp_rdp_recv failed
12月 05 10:59:27  xrdp[88005]: [ERROR] xrdp_process_data_in: xrdp_process_loop failed
12月 05 10:59:27  xrdp[88005]: [ERROR] SSL_shutdown: I/O error

**Comment by 720lhy** [comment]
![image](https://github.com/user-attachments/assets/930858d3-33f3-4a83-a353-defc9747e91e)

**Comment by matt335672** [comment]
There's not nearly enough information in here to work out what your problem is. Please raise an issue and fill in the form.
